### ISSUE TITLE
Support PHP 7

### ISSUE THREAD

**Issue by Mahmoudz** [LaraDock/laradock#22]

**Comment by AminMkh** [comment]
I like the idea, I use [PHP's image](https://hub.docker.com/_/php/) The PHP7, Apache, Ubuntu version.

```
FROM php:7-apache

# installing required stuff
RUN apt-get update \
    && apt-get install -y unzip libaio-dev libmcrypt-dev git \
    && apt-get clean -y

# PHP extensions
RUN \
    docker-php-ext-configure pdo_mysql --with-pdo-mysql=mysqlnd \
    && docker-php-ext-configure mysqli --with-mysqli=mysqlnd \
    && docker-php-ext-install pdo_mysql \
    && docker-php-ext-install mbstring \
    && docker-php-ext-install mcrypt

# xdebug, if you want to debug
RUN pecl install xdebug

# PHP composer
RUN curl -sS https://getcomposer.org/installer | php -- --version=$(curl -s https://api.github.com/repos/composer/composer/releases/latest | grep -oP '"tag_name": "\K[^"]+')  --install-dir=/usr/bin --filename=composer


# apache configurations, mod rewrite
RUN ln -s /etc/apache2/mods-available/rewrite.load /etc/apache2/mods-enabled/rewrite.load
```

**Comment by AminMkh** [comment]
and maybe sharing the PHP configurations folder + MySQL in case geeks wanted to tune stuff.
```
Web:
  volumes:
    - ./conf/php:/usr/local/etc/php/
db:
  volumes:
    - ./conf/mysql:/etc/mysql/
```

**Comment by Mahmoudz** [comment]
@AminMkh that looks very good. However I prefer `NGINX` instead of `Apache`, and all the current images are based on `phusion/baseimage`. So I wish if you can contribute with `PHP7` + `NGINX` on `phusion/baseimage`. Thanks.

**Comment by Mahmoudz** [comment]
Finally this feature is now available, check the release version `1.0.0`
https://github.com/LaraDock/laradock/releases/tag/v1.0.0
